gpu.Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG BASE_TAG=staging
 
 FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu18.04 AS nvidia
-FROM gcr.io/kaggle-images/python-tensorflow-whl:2.1.0-py37 as tensorflow_whl
+FROM gcr.io/kaggle-images/python-tensorflow-whl:2.1.0-py37-2 as tensorflow_whl
```

Comment:
Where does the name "py37-2" come from? I only saw it in the other PR branch in the change logs, but I'm guessing that naming is set via a convention somewhere?

Reply:
The convention is `{TF_VERSION}-py{PYTHON-VERSION}{optional version if multiple images are built for the same tf and python version}`